src/plugins/intel_npu/tools/protopipe/src/simulation/ov_layers_reader.cpp

```diff
@@ -44,12 +44,17 @@ static ov::element::Type toElementType(int cvdepth) {
     throw std::logic_error("Failed to convert opencv depth to ov::element::Type");
 }
 
-static std::vector<int> toDims(const std::vector<size_t>& sz_vec) {
+static std::vector<int> toDims(const ov::PartialShape& partialShape) {
```

Comment:
The style is `snake_case` for the entire project

Reply:
Done